compiler/testData/codegen/box/properties/const/anotherFile.kt

```diff
@@ -0,0 +1,7 @@
+// FILE: 1.kt
+// File names are important! This file should come before the other one
```

Comment:
There are also other tests that fail with different orderings, e.g.

```
// FILE: 1.kt
fun box() = A.f()
// FILE: 2.kt
object A {
    @JvmStatic
    fun f(x: String = "OK") = x
}
```

If having all lowerings do the right thing on both lowered and unlowered inputs is still a goal, perhaps all multi-file tests should run twice (once with the original file names, and once with files renamed to be in reverse order)?

Reply:
I like this idea. My only concern is that it'd increase the IR tests run time by a few hundred test cases, where majority of those cases supposedly don't test anything useful. Maybe an explicit directive in the test source would be a better approach